release_collection.py

```diff
@@ -168,6 +169,28 @@ def process_ignore_and_lint_files(args, coll_dir):
         yaml.safe_dump(ansible_lint, open(os.path.join(coll_dir, ".ansible-lint"), "w"))
 
 
+def append_changelog(args, rolename, cur_ref, new_ref):
```

Comment:
This function isn't appending anything, is it?  It looks like it is doing `get_role_changelog` or something like that.

Reply:
That's true. :p Sorry, I changed the original plan and forgot to adjust the function name...
Thanks!